src/components/theme/App/App.jsx

```diff
@@ -89,21 +74,6 @@ class App extends Component {
     }
   }
 
-  /**
-   * ComponentDidCatch
-   * @method ComponentDidCatch
-   * @param {string} error  The error
-   * @param {string} info The info
-   * @returns {undefined}
-   */
-  componentDidCatch(error, info) {
```

Comment:
@zotya we can't remove this completely, since it has the default error handler. Removing the Sentry part is fine, but not the whole thing.

Reply:
@sneridagh Adding back the default error handler actually solved my issue with guillotina tests.